pkg/contracts/src/CVStrategy.sol

```diff
@@ -794,7 +804,7 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
         // console.log("maxRatio * funds", maxRatio * funds);
         // console.log("_requestedAmount * D", _requestedAmount * D);
 
-        if (maxRatio * funds <= _requestedAmount * D) {
```

Comment:
you can remove the uint256 funds variable and change to poolAmount in the formula